### ISSUE TITLE
How to use FileStorage in Memory mode.

### ISSUE THREAD

**Issue by turowicz** [shimat/opencvsharp#707]
Hi I would like to switch from writing YML calibration files to disk to saving them in a database and therefore would like to use the serializer to write/read from memory.

Can I something like this:

```
using (var fs = new FileStorage(null, Mode.Write | Mode.FormatYaml | Mode.Memory))
{
    fs.Write("camera", _cameraMatrix);
    fs.Write("distortion", _distortionMatrix);
    
    SaveToDatabase(fs.ReleaseAndGetString());
}
```

The same goes from reading the object from memory string.

**Comment by turowicz** [comment]
cc @shimat

**Comment by turowicz** [comment]
Actually the above code works but I can't figure out how to read from an in-memory string.

**Comment by shimat** [comment]
```c#
// Write
string yaml;
using (var fs = new FileStorage(null, Mode.Write | Mode.FormatYaml | Mode.Memory))
{
    yaml = fs.ReleaseAndGetString();
}

// Read
using (var fs = new FileStorage(yaml, Mode.Read | Mode.Memory))
{
}
```

However, ReleaseAndGetString has a bug that prevents it from returning strings longer than 65536 characters. Output string becomes so long when we add images to FileStorage. I fixed the bug in #708.

**Comment by turowicz** [comment]
@shimat thanks!
